js/renovation/ui/form/form.tsx

```diff
@@ -1,24 +1,31 @@
 import {
-  Component,
-  JSXComponent,
+  Component, JSXComponent,
 } from '@devextreme-generator/declarations';
 
 import { FormProps } from './form_props';
 
 import { combineClasses } from '../../utils/combine_classes';
 import { Widget } from '../common/widget';
+import { FormLayoutManager } from './form_layout_manager';
 import { Scrollable } from '../scroll_view/scrollable';
 
+const renderLayoutManager = (isRoot: boolean): JSX.Element => (
```

Comment:
isRoot is not used
Let's remove it or use it in this PR

Reply:
fixed